controllers/product.js

```diff
@@ -0,0 +1,62 @@
+import Product from '../models/product.js';
+
+export const createProduct = async (req, res) => {
+    const { name, color, length, price, images } = req.body;
+
+    if (!name || !color || !length || !price || !images) {
+        return res.status(403).json({error: 'Please provide all required fields.'})
```

Comment:
I would like to use the `StatusCodes` for the 'http-status-codes` library we installed. So instead of saying 200 or 403. It would be res.status(StatusCodes.OK) or res.status(StatusCodes.BAD_REQUEST). 

You can import on top of page like this `import { StatusCodes } from 'http-status-codes';`